src/main/java/menu/view/OutputView.java

```diff
@@ -0,0 +1,57 @@
+package menu.view;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import menu.domain.dto.CategoriesDto;
+import menu.domain.dto.CategoryDto;
+import menu.domain.dto.CoachDto;
+import menu.domain.dto.CoachesDto;
+import menu.domain.dto.RecommendedMenuDto;
+import menu.domain.dto.RecommendedResultDto;
+
+public class OutputView {
+    private static final String RESULT_MESSAGE = System.lineSeparator() + "메뉴 추천 결과입니다.";
+    private static final String WEEKDAYS_MESSAGE = "[ 구분 | 월요일 | 화요일 | 수요일 | 목요일 | 금요일 ]";
+    private static final String FORMAT = "[ %s | %s ]";
+    private static final String CATEGORY = "카테고리";
```

Comment:
```suggestion
"""
메뉴 추천 결과입니다.
[ 구분 | 월요일 | 화요일 | 수요일 | 목요일 | 금요일 ]
[ 카테고리 | %s ]
"""
```

위 코드처럼 작성하면 코드가 훨씬 줄어들 것 같습니다